### ISSUE TITLE
fix stata documentation in vignette

### ISSUE THREAD

**Issue by josherrickson** [markmfredrickson/optmatch#158]
When importing the match back into Stata, suggest something like

```
match <- as.character(fullmatch(...))
```
Then
```
import delim using ...., stringcols(2) 
```

To force Stata to treat as factor. Maybe there's a better solution?

**Comment by josherrickson** [comment]
Also, `write.dta` should be `write.csv`.

**Comment by josherrickson** [comment]
Updated the vignette using haven to write & read the Stata file. With the fix to #159, this is now very easy.
